src/components/group-common.js

```diff
@@ -25,17 +24,17 @@ const styles = {
   }
 }
 
-export const NewUserCard = pure(({ onClick }) => {
+export const NewUserCard = ({ onClick }) => {
   return h(Clickable, {
     style: Style.cardList.shortCreateCard,
     onClick
   }, [
     div(['Add a User']),
     icon('plus-circle', { style: { marginTop: '0.5rem' }, size: 21 })
   ])
-})
+}
 
-export const MemberCard = pure(({ member: { email, roles }, adminCanEdit, onEdit, onDelete, adminLabel, userLabel }) => {
+export const MemberCard = ({ member: { email, roles }, adminCanEdit, onEdit, onDelete, adminLabel, userLabel }) => {
```

Comment:
FYI, `pure` was still having an effect, by adding a wrapper with a `shouldComponentUpdate` check. The preferred way to do that now is with the built-in `memo`. However, I suspect that these usages weren't providing much value and it's safe to remove them.